system/lib/compiler-rt/sanitizer-common.symbols

```diff
@@ -0,0 +1,2058 @@
+
+sanitizer_stoptheworld_linux_libcdep.cc.o:
+
+sanitizer_symbolizer_mac.cc.o:
+
+sanitizer_procmaps_mac.cc.o:
+
+sanitizer_libc.cc.o:
+-------- d .L.str
+-------- d .L.str.2
+-------- d .L.str.3
+         U _ZN11__sanitizer11CheckFailedEPKciS1_yy
+-------- T _ZN11__sanitizer11mem_is_zeroEPKcm
+         U _ZN11__sanitizer13InternalAllocEmPNS_28SizeClassAllocatorLocalCacheINS_20SizeClassAllocator32INS_4AP32EEEEEm
+-------- T _ZN11__sanitizer14internal_atollEPKc
+-------- T _ZN11__sanitizer15internal_memchrEPKvim
+-------- T _ZN11__sanitizer15internal_memcmpEPKvS1_m
+-------- T _ZN11__sanitizer15internal_memcpyEPvPKvm
+-------- T _ZN11__sanitizer15internal_memsetEPvim
+-------- T _ZN11__sanitizer15internal_strchrEPKci
+-------- T _ZN11__sanitizer15internal_strcmpEPKcS1_
+-------- T _ZN11__sanitizer15internal_strdupEPKc
+-------- T _ZN11__sanitizer15internal_strlenEPKc
+-------- T _ZN11__sanitizer15internal_strstrEPKcS1_
+-------- T _ZN11__sanitizer16internal_memmoveEPvPKvm
+-------- T _ZN11__sanitizer16internal_memrchrEPKvim
+-------- T _ZN11__sanitizer16internal_strcspnEPKcS1_
+-------- T _ZN11__sanitizer16internal_strlcatEPcPKcm
+-------- T _ZN11__sanitizer16internal_strlcpyEPcPKcm
+-------- T _ZN11__sanitizer16internal_strncatEPcPKcm
+-------- T _ZN11__sanitizer16internal_strncmpEPKcS1_m
+-------- T _ZN11__sanitizer16internal_strncpyEPcPKcm
+-------- T _ZN11__sanitizer16internal_strnlenEPKcm
+-------- T _ZN11__sanitizer16internal_strrchrEPKci
+-------- T _ZN11__sanitizer18internal_strchrnulEPKci
+-------- T _ZN11__sanitizer23internal_simple_strtollEPKcPS1_i
+
+sanitizer_allocator.cc.o:
+-------- d .L.str
```

Comment:
I don't see these local symbols on the other `.symbols` file.  Is there something we are doing differently here?

Reply:
I am not sure how these files are generated. I just did `llvm-nm` and replaced the addresses with `-` with `perl`.